host-bin/enter-chroot

```diff
@@ -676,6 +679,12 @@ else
     else
         # Escape out the command
         cmd="export SHELL='$CHROOTSHELL';"
+        if [ -n "$TMPXMETHOD" ]; then
+-            case "$TMPXMETHOD" in
+-                xiwi|xorg|xephyr) cmd="$cmd export XMETHOD='$TMPXMETHOD';" ;;
+-                *) echo "WARNING: XMETHOD '$TMPXMETHOD' invalid so not set." ;;
```

Comment:
Please validate the parameter way earlier (where the rest of the parameters are validated), and also check if the desired target is actually installed (using .crouton-targets).